### ISSUE TITLE
Builds failing at tmpfs mounting step

### ISSUE THREAD

**Issue by sanjayankur31** [fedora-copr/copr#2916]
I'm seeing all my builds on COPR fail with errors of this form:

```
ERROR: Command failed: 
 # mount -n -t tmpfs -o mode=0755 -o nr_inodes=0 -o size=140g mock_chroot_tmpfs /var/lib/mock/fedora-rawhide-x86_64-1694966985.517941/root
```

Full example log:
https://download.copr.fedorainfracloud.org/results/@neurofedora/neurofedora-extra/fedora-rawhide-x86_64/06412281-python-modelspec/builder-live.log.gz

Example build:
https://copr.fedorainfracloud.org/coprs/g/neurofedora/neurofedora-extra/build/6412281/

Could someone please take a look when possible?

**Comment by r4sas** [comment]
I have the same error and I somehow solved it by changing the "Mock bootstrap" option to "Use default configuration from mock-core-configs.rpm" in project options.

**Comment by praiskup** [comment]
Thank you both (for the report and for finding a workaround).  Indeed, there's a bug in Mock 5.1
which causes this, and "bootstrap ON" actually accidentally works-around it.

**Comment by praiskup** [comment]
Also reported on IRC several times and here:
https://lists.fedorahosted.org/archives/list/copr-devel@lists.fedorahosted.org/thread/AQTPYXTGQIWJA5A5GDQEWBGHKYYL4AIY/

**Comment by praiskup** [comment]
Fixed in Fedora Copr production (Mock 5.1.1 deployed).
